packages/shortest/src/cli/commands/init.ts

```diff
@@ -209,3 +474,37 @@ export const getInstallCmd = async () => {
     toString: () => cmdString,
   };
 };
+
+export const taskWithCustomLogOutput = <T>(
+  task: TaskWrapper<Ctx, typeof DefaultRenderer, typeof SimpleRenderer>,
+  callback: () => Promise<T> | T,
+): Promise<T> => {
+  const log = getLogger();
+  const originalFormat = log.config.format;
+  const originalLevel = log.config.level;
+
+  log.config.level = "trace";
+  const streamAdapter = new Writable({
+    write(chunk, _encoding, callback) {
+      task.stdout().write(chunk.toString());
+      callback();
+    },
+  });
+
+  log.setOutputStream(streamAdapter);
```

Comment:
Passing the output of our logger to the task's stdout so we can display the logs below the interactive list.